src/test/java/core/basesyntax/service/impl/SubtractOperationHandlerImplTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SubtractOperationHandlerImplTest {
+    private static final String KEY_APPLE = "apple";
+
+    @BeforeClass
+    public static void beforeClass() {
+        FruitDao fruitDao = new FruitDaoImpl();
+        Map<FruitTransaction.Operation, OperationHandler> handlerMap = new HashMap<>();
+        handlerMap.put(FruitTransaction.Operation.SUPPLY, new AddOperationHandlerImpl(fruitDao));
+        handlerMap.put(FruitTransaction.Operation.PURCHASE,
+                new SubtractOperationHandlerImpl(fruitDao));
+    }
```

Comment:
```suggestion
    public static void beforeClass() {
        FruitDao fruitDao = new FruitDaoImpl();
    }
```
looks like we don't need this
check this in other tests